docs/docs/widgets/steps.md

```diff
@@ -36,25 +38,29 @@ This property when disabled will disable selection of steps.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
 | <div style={{ width:"100px"}}> Event </div> | <div style={{ width:"100px"}}> Description </div> |
 |:------------------|:---------------------|
 | On select | This event is fired whenever the user selects any step. |
+| On change | This event triggers when the user changes the text in the input. |
```

Comment:
Hey @prathamesh424, It looks like you have directly copied the events table from the text input component while you were just expected to refer it and update the content according to the steps component and please check if this events are even available for step component in tooljet app builder.

Please update this table in both versions.

Reply:
where I can find the events  so will add containtent based on that

Reply:
Hey @prathamesh424,

You just need to update the description of the "on select" event. The reference is provided to help you understand the sentence structure used in other components.